src/Symfony/Component/Translation/Translatable.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation;
+
+/**
+ * @author Nate Wiebe <nate@northern.co>
+ */
+final class Translatable
+{
+    private $messageId;
+
+    private $parameters;
+
+    private $domain;
+
+    public function __construct(string $messageId, array $parameters = [], string $domain = 'messages')
+    {
+        $this->messageId = $messageId;
+        $this->parameters = $parameters;
+        $this->domain = $domain;
+    }
+
+    /**
+     * @return string The message id (may also be an object that can be cast to string)
```

Comment:
You can remove the whole phpdoc (we tend to not add any phpdocs when not needed).

Reply:
in case of a stringable object, it would already be casted to string in __construct.

Reply:
Updated